.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,21 @@
+resources:
+  containers:
+  - container: centos7
+    image: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
+
+phases:
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Linux
+    buildScript: ./build.sh
+    queue:
+      container: centos7
+
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Windows_NT
```

Comment:
>Windows_NT [](start = 10, length = 10)

I obviously don't have knowleage regarding build system, but from what I see we have 4 nodes in VSTS  where Windows_NT is not running, and also Windows builds on jenkins.
Is it intended?

Reply:
So the jenkins Windows builds will stop showing up after I merge this PR. They keep showing because it takes the checked-in netci.groovy file instead of taking the one in this PR.

About the VSTS nodes, you can see here: https://dotnet.visualstudio.com/public/_build/results?buildId=7047&view=logs that the new Build Def, will run Linux and Windows in both Release and Debug mode simultaneously.

Reply:
Got it, thank you for clarification.